L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc

```diff
@@ -317,10 +317,23 @@ for(int SectIndex=0;SectIndex<NUM_SECTORS;SectIndex++){//perform TF on all 12 se
 			}
 
 		}
+		
+		
+		int mode = 0;
+		if(tempTrack.rank & 32)
```

Comment:
A case statement won't work here, as it is testing each bit independently... dropping this one.